projects/ccf-shared/src/lib/components/organ-selector/organ-selector.component.ts

```diff
@@ -121,6 +121,7 @@ export class OrganSelectorComponent implements AfterViewInit, OnDestroy {
    * Set resize sensor on carousel
    */
   ngAfterViewInit(): void {
+    this.selectedOrgans = this.selectedOrgans.slice(1);
```

Comment:
What is this?

Reply:
It's to deselect the first item by default. Should I do it some other way?

Reply:
Why would you do that?